src/app/login/success/page.tsx

```
@@ -28,19 +30,22 @@ const Page = () => {
           case 'ROLE_REPORTED':
             router.push('/blackhole');
             break;
+          case 'ROLE_ADMIN':
+            router.push('/admin');
           default:
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)